CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 
 ## [Unreleased]
 
+## [2.0.3] - 2018-09-14
+### New Feature
```

Comment:
When [adding a new feature](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md#adding-a-new-feature) we use the `### Added` header

Reply:
Feel free to update it.